lib/qx/tool/cli/commands/contrib/Install.js

```diff
@@ -151,6 +151,9 @@ qx.Class.define("qx.tool.cli.commands.contrib.Install", {
       let library_elem = null;
       for (let {namespace, info, path:filename} of release_data.manifests) {
         // does the repository name already exist?
+        if (!namespace) {
+          throw new qx.tool.cli.Utils.UserError("Tag 'namespace' not found. Please run qx contrib update!");
```

Comment:
maybe implicit update instead? although this will lead to infinite recursion until the cache is updated for the first time, so maybe this should be done in a later PR.

Reply:
Implicit update will not work - place to Deep in the program. And after Manual update this is fixed for ever.